Jenkinsfile

```diff
@@ -24,7 +24,7 @@ def runTests(Map params = [:]) {
             }
             // Needed for correct computation of JenkinsHome in RunMojo#execute.
             withEnv(['JENKINS_HOME=', 'HUDSON_HOME=']) {
-              infra.runMaven(args, params['jdk'])
+              infra.runMaven(args, params['jdk'], null, null, false)
```

Comment:
Does the build pass or fail with this hunk reverted?

Reply:
The build with only ACP disabled + the settings.xml merged passes (when replayed otherwise my Jenkinsfile change is not taken in account).

Reply:
Right. What about the build with this hunk reverted?

Reply:
I thought it failed but as I'm not so sure, let's retry. I've removed this hunk let's see

Reply:
I confirm that https://github.com/jenkinsci/maven-hpi-plugin/pull/537/commits/678094ac1aa1d40e85826bd0192db3c1ec25b226  fails (e.g. only "merging settings.xml" but no ACP).

Reply:
> I confirm that [678094a](https://github.com/jenkinsci/maven-hpi-plugin/commit/678094ac1aa1d40e85826bd0192db3c1ec25b226) fails (e.g. only "merging settings.xml" but no ACP) - build https://ci.jenkins.io/job/Plugins/job/maven-hpi-plugin/job/PR-537/14/

Hence my proposal: Add back the hunk which disable ACP for the build. It consumes a bit of bandwidth but ensures that Integration Tests are run without any problem.